custom/icds_reports/ucr/reports/ls/ls_v2_report_lbw_pre_term.json

```diff
@@ -0,0 +1,180 @@
+{
+  "domains": [
+    "icds-dashboard-qa",
+    "sankalp_cas",
+    "zohaib-sandbox",
+    "akshita-sandbox",
+    "sunaina-sandbox",
+    "laurence-project-1",
+    "jessica-icds-cas",
+    "marissa-test",
+    "derek-icds-sandbox",
+    "priyanka-app",
+    "shrena-dev",
+    "aparatest",
+    "reach-sandbox",
+    "reach-dashboard-qa",
+    "reach-test",
+    "icds-test",
+    "icds-sql",
+    "icds-cas",
+    "cas-lab",
+    "icds-cas-sandbox"
+  ],
+  "server_environment": [
+    "india",
+    "icds"
+  ],
+  "report_id": "ls_v2_report_lbw_pre_term",
+  "data_source_table": "static-child_health_cases",
+  "config": {
+    "title": "UCR V2 - LS Report - Low Birth Weight & Pre Term",
+    "description": "",
+    "visible": false,
+    "aggregation_columns": [
+      "doc_id",
+      "supervisor_id"
+    ],
+    "filters": [
+      {
+        "compare_as_string": false,
+        "datatype": "string",
+        "required": false,
+        "display": "Date of Birth",
+        "field": "dob",
+        "type": "date",
+        "slug": "dob"
+      },
+      {
+        "compare_as_string": false,
```

Comment:
this might want to be a `pre` filter rather than a choice filter with only one option

Reply:
I will pass this feedback on, but this isn't really a filter with one option, since it also has "Show All." It's a little strange it only has "yes" and not "no", but it does seem possible that users of this report don't care to filter for infants who aren't low birth weight.

Reply:
That sounds even more confusing. Its a report called "preterm or low birth weight" and the data inside is just name, dob, and sex. Which means if "all" is selected, then you cant tell which children were preterm or low birth weight. It is just a list of every child, without distinguishing characteristics.

Reply:
Hmm, I see your point better now. Will pass on this feedback a bit more aggressively.